src/components/SearchNftsTable.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <table class="search-nfts-table">
+    <thead>
+      <tr>
+        <th>
+          Symbol
+          <hint-tooltip>
+            {{ nftsHints.symbol }}
+          </hint-tooltip>
+        </th>
```

Comment:
```suggestion
```

I would remove the "Symbol" column to keep it consistent with the NFTs page where we don't have it.